ui/app/components/ui/popover/index.scss

```diff
@@ -85,6 +85,7 @@
     justify-content: flex-start;
     align-items: stretch;
     align-content: stretch;
+    padding-left: 6px; // to match webkit-scrollbar width
```

Comment:
I suspect this would make things look inconsistent on Firefox, as the `::-webkit-scrollbar` selector doesn't work on Firefox.

Maybe this can be fixed separately? We could also find some way of extending the row border through the scrollbar space when it isn't present, rather than adding padding to match on the left.

Reply:
Yeah, I removed that. Also more consistent with designs.

I changed the `overflow-y` in that file to `auto` instead of `scroll`. Tested with Chrome and Firefox and seems to work fine. Was there a particular reason for the hard `scroll` that you can recall?

I also changed the the borders to `border-top` per row and no `border-bottom` of the above element to get rid of the overflowing bottom border.